aws/auto-scaling.tf

```diff
@@ -22,6 +22,9 @@ resource "aws_autoscaling_group" "bastion" {
   # THis needs to match the Launch Configuration.
   lifecycle {
     create_before_destroy = true
+
+    # Allow end user to attach a load balancer with `aws_autoscaling_attachment`.
+    ignore_changes = [load_balancers, target_group_arns]
```

Comment:
Would you say more about the situation that requires this `ignore_changes`, or perhaps a snippet from `terraform plan` that describes how the load balancer attachment requires this bastion module to need this ignore stanza?

Reply:
That's recommanded in [autoscaling_attachment resource docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/autoscaling_attachment#with-an-autoscaling-group-resource). I still seems confused to me though, but from the docs it seems we should have those ignore_changes here to avoid issues with attaching a load balancer.

Reply:
Aside from the documentation recommending that, what happens if you `terraform plan` when using your feature **without** the `ignore_changes`?